Automattic/wp-calypso
# Update the Anti-Spam card stats copy

#### Changes proposed in this Pull Request

Update the copy in the anti-spam card on Pricing page

#### Testing instructions

* checkout and boot this branch as Jetpack Cloud
* verify that the last line of details in the Anti-spam card now says "10K API calls/mo" and not 'Advanced stats'
